bbb-exporter/collector.py

```diff
@@ -284,6 +295,20 @@ def _get_participant_count_by_client(meetings):
 
         return p_by_c
 
+    @staticmethod
+    def _get_participant_count_by_metadata(meetings, metadata_key = 'bbb-origin-server-name'):
+        p_by_m = defaultdict(int)
+        for meeting in meetings:
+            participants = int(meeting['participantCount'])
+            if participants == 0:
+                continue
+            if not meeting.get("metadata") or not meeting['metadata'].get(metadata_key):
+                p_by_m['_no_metadata'] += participants
```

Comment:
Add to the empty key instead of the magic key `_no_metadata`:
```python3
p_by_m[''] += participants
```

This way the Prometheus library should generate an empty label, which is more semantically correct than a hardcoded magic string.

Reply:
Thanks, I thought about that and decided against; I am happy with you voting for it and changed it accordingly :)